build-tools/packages/build-tools/src/repoPolicyCheck/handlers/npmPackages.ts

```diff
@@ -276,7 +276,7 @@ export const handlers: Handler[] = [
 				json.homepage = homepage;
 			}
 
-			writeFile(file, JSON.stringify(sortPackageJson(json), undefined, 2) + newline);
+			updatePackageJsonFile(path.dirname(file));
```

Comment:
this change is wrong. It will just discard all the changes made above. you need to put the abovecode in the callback you omitted.

Reply:
facepalm... thanks.